### PR TITLE
PLAD-109  As an user, I want to be announced when I received a match

### PR DESCRIPTION
**Pull request by AliceCasali**

### CODE REVIEW

File: backend/DAO/studentDAO.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = {
         
         // This one is very similar to SQL
         let result = await database('student')
-            .select('id', 'name','surname')
+            .select('id', 'name','surname', 'imgurl')
```
Comment (Fabs97) [inline backend/DAO/studentDAO.js:15]: I don't think this is needed here @AliceCasali

File: frontend/lib/main.dart [hunk 2]
```diff
@@ -12,6 +12,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
```
Comment (Fabs97) [inline frontend/lib/main.dart:15]: Uhm, eh? @AliceCasali

File: frontend/lib/widgets/match_alert.dart [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                      color: Colors.white,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
```
Comment (Fabs97) [inline frontend/lib/widgets/match_alert.dart:137]: @AliceCasali maybe some spacing with MainAxisAlignment.spaceEvenly between the elements in the main Column could give a nicer feeling of the card, rn everything is a bit squished together

**Review by Fabs97 — CHANGES_REQUESTED** [review]

File: frontend/lib/main.dart (hunk 2 above)
Comment (AliceCasali) [reply frontend/lib/main.dart:15]: This is needed in order to remove the debug icon in the top right corner of the screen. I cannot see my close icon without this command

**Review by Fabs97 — APPROVED** [review]

**Pull request merged**